src/loaders/compute_tools/bbmap/bbmap.py

```diff
@@ -0,0 +1,66 @@
+"""
+Run BBMap tool on a set of fna files.
+
+This tool serves a distinct purpose separate from collection tools; instead, it is suited for CDM work.
+Therefore, the parser program is not compatible with data generated by this tool.
+
+"""
+import time
+from pathlib import Path
+
+from src.loaders.common.loader_common_names import TOOL_METADATA
+from src.loaders.compute_tools.tool_common import ToolRunner, run_command, create_tool_metadata
+
+
+def _run_bbmap_single(
+        tool_safe_data_id: str,
+        data_id: str,
+        source_file: Path,
+        output_dir: Path,
+        threads_per_tool_run: int,
+        debug: bool) -> None:
+    start = time.time()
+    print(f'Start executing BBMap for {data_id}')
+
+    metadata_file = output_dir / TOOL_METADATA
+    if metadata_file.exists():
+        print(f"Skipping {source_file} as it has already been processed.")
+        return
+
+    command = [
+        'stats.sh',
```

Comment:
My understanding is that bbtools does a lot of stuff. We're not interested in anything except this?

Reply:
that's the only thing the science need for now.

Reply:
Would it be difficult to handle if we needed to use other bbtools functionality?

Reply:
I don't see an alternative. With the current setup we would have to build a new image for even an argument update.

Reply:
Hmm, we should discuss this in the standup tomorrow. 

Anyway, nothing to be done for now I guess

:+1: